Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM tiangolo/uwsgi-nginx:python3.8
+
+ENV UWSGI_INI /srv/www/yogsite/uwsgi.ini
+
+COPY . /srv/www/yogsite
+COPY nginx.conf /etc/nginx/conf.d/
```

Comment:
I would rename nginx.conf so something more unique, also shouldn't this go into sites-enabled

I think a symlink would be better than a copy too